File: uncategorized/1887_reduction_operations_to_make_the_array_elements_equal.go
func reductionOperations(nums []int) int {

    sort.Ints(nums)

    ans := 0

    for i :=len(nums)-2; i>-1; i-- {
        if nums[i] != nums[i+1] {
            ans += len(nums) - (i+1)
        }
    }

    return ans

}

File: uncategorized/1887_reduction_operations_to_make_the_array_elements_equal.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var reductionOperations = function(nums) {

    nums.sort((a,b) => b - a)

    let ans = 0

    for(let i=1; i<nums.length; i++){
        if(nums[i] !== nums[i-1]) ans += i
    }

    return ans

};